src/test/kotlin/org/eclipse/uprotocol/cloudevent/serialize/CloudEventToJsonSerializerTest.kt

```diff
@@ -58,13 +57,14 @@ internal class CloudEventToJsonSerializerTest {
             .withExtension("ttl", 3)
             .withExtension("priority", "CS1")
         val cloudEvent: CloudEvent = cloudEventBuilder.build()
+        //cloudEvent.getAttribute("traceparent")
```

Comment:
do we want to keep this commented out? or maybe we should just remove this?

Reply:
forgot to clean up